src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dataArray = date.split(fromFormat[3]);
+  const result = {};
+  const resultArray = [];
+
+  for (let i = 0; i < 3; i++) {
+    result[fromFormat[i]] = dataArray[i];
+  }
+
+  function recordTypeYear(number) {
+    switch (number) {
+      case 2:
+        return result['YYYY'].slice(2);
+      default:
+        return (century(result['YY']) + result['YY']);
+    }
+  }
+
+  function century(number) {
+    if (number < 30) {
+      return 20;
+    } else {
+      return 19;
+    }
```

Comment:
```suggestion
    if (number < 30) {
      return 20;
    }
    
    return 19;
```

if you return immediately you can avoid `else`